Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu:18.10
```

We do not need to use such big default image, we can probably go with a minimal based on node.

```suggestion
FROM node:12-alpine
```